components/channel_layout/center_channel/index.ts

```diff
@@ -31,13 +38,36 @@ const mapStateToProps = (state: GlobalState, ownProps: Props) => {
         channelName = getRedirectChannelNameForTeam(state, team!.id);
     }
     const lastChannelPath = `${ownProps.match.url}/channels/${channelName}`;
+    const currentUserId = getCurrentUserId(state);
     return {
         lastChannelPath,
         lhsOpen: getIsLhsOpen(state),
         rhsOpen: getIsRhsOpen(state),
         rhsMenuOpen: getIsRhsMenuOpen(state),
         isCollapsedThreadsEnabled: isCollapsedThreadsEnabled(state),
+        currentUserId,
```

Comment:
Nit: since we aren't we re-using this, can we just have it in-line?
```suggestion
        currentUserId: getCurrentUserId(state),
```